requires.py

```diff
@@ -42,4 +42,5 @@ def get_connection_info(self):
         conv = self.conversation()
         ip = conv.get_remote('private-address')
         port = conv.get_remote('port')
-        return {"ip": ip, "port": port}
+        controllers = conv.get_remote('controllers')
+        return {"ip": ip, "port": port, "controllers": controllers}
```

Comment:
minor knit here, but since we're now passing more than connection info, i'm gonna update this function name to `get_cwr_info`.